### PR TITLE
Manage version in comfig.cfg

### PR DESCRIPTION
**Pull request by micwoj92**
this way echo.cfg doesn't have to be updated

Tested this vpk and it works.

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -1661,3 +1661,6 @@ alias game_overrides_once game_overrides_once_c
 block_game_overrides_once
 
 alias version_comfig"echo mastercomfig version: 8.102.4a6 | June 18 2020"
+alias version_c"echo mastercomfig 8.102.4a6 loaded"
+alias date_c"echo June 18 2020"
```
[the remark this answers is not in the document]
Comment (micwoj92) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: My proposal is to use ISO 8601

[the remark this answers is not in the document]
Comment (mastercoms) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: My problem with numeric date formats is that they can cause confusion between users in the United States and other users. So a date format with the month as a word is preferred.

[the remark this answers is not in the document]
Comment (micwoj92) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: Oh I understand. But still, is it fine or raher add comma as the bot suggests?

[the remark this answers is not in the document]
Comment (mastercoms) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: I guess adding a comma is good.

[the remark this answers is not in the document]
Comment (micwoj92) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: added

**Review by mastercoms — CHANGES_REQUESTED** [review]
I don't like how the spacing was removed.

**Comment by micwoj92** [comment]
When you alias something to echo then the echo doesn't output the first space, i tried with different whitespace characters and it either didn't work or gave "???" error signs, so in my opinion this was best workaround. Feel free to close if you don't like it.

**Comment by mastercoms** [comment]
Yeah I think I can manage using the different files for version. Thanks for trying.

**Pull request closed without merging**